scripts/build/index.js

```diff
@@ -244,8 +244,7 @@ const buildServiceImages = async () => {
 
   console.log('working directory', workingDir);
   for (const service of versions.SERVICES) {
-    await exec('cp', ['-r', `${workingDir}/node_modules`, service]);
-    await exec('cp', ['-r', './shared-libs', service]);
```

Comment:
These 2 lines leave a `node-module` and `shared-libs` folders inside `api` and `sendinel` directories. Then git shows like +600 new files to add in the diff.  
It doesn't need to copy the `shared-libs`. While the `node_modules` is already in the git ignore. 

@garethbowen is this okay? I wonder if I'm not seeing something about copying `shared-libs`.

Reply:
This was an attempt to simplify the Dockerfile and get development closer to prod, but clearly this was a step too far. Thanks for finding it!